templates/zerver/help/edit-your-profile.md

```diff
@@ -12,16 +12,12 @@ fill them out.
 
 {start_tabs}
 
-{settings_tab|your-account}
+{settings_tab|profile}
 
 1. Edit the fields under **Profile**.
 
 {end_tabs}
 
-!!! warn ""
-    Note: The **Profile** section will only appear if an organization
-    administrator has added custom profile fields.
-
```

Comment:
I am not sure what to write here as we have a single `Profile` section which contains name field also, so just removed it for now.

Reply:
Yeah, I think deleting this is correct.